package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "@asyncapi/fluffy-robot",
+  "version": "0.0.1",
+  "type": "commonjs",
+  "description": "The asyncapi command line library to test your app at scale.",
+  "main": "src/bin/cli.js",
+  "engines": {
+    "node": "14.15.5"
+  },
+  "scripts": {
+    "test": "echo \"No test specified\"",
+    "release": "semantic-release",
+    "docker:build": "docker build -t asyncapi/fluffy-robot:latest .",
+    "get:version": "echo $npm_package_version",
+    "get:name": "echo $npm_package_name",
+    "lint": "eslint --max-warnings 0 --config .eslintrc .",
+    "generate:assets": "echo \"Nothing to generate\"",
+    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION"
+  },
+  "keywords": [
+    "asyncapi",
+    "simulation"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": {
+    "email": "nektariosfifes12@gmail.com",
+    "name": "NektariosFifes",
+    "url": "https://github.com/NektariosFifes"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@asyncapi/parser": "^1.4.1",
+    "ajv": "^8.6.0",
+    "chalk": "^4.1.0",
+    "commander": "^7.2.0",
+    "install": "^0.13.0",
+    "npm": "^7.17.0"
```

Comment:
What are you using these two dependencies for? 🤔

Reply:
i think they installed because i used npm install with another node version.

Reply:
i guess it is for using a specific npm version installed in node_modules

Reply:
could these be usefull?

Reply:
Only if they are used, otherwise no, anything that is never used should never be committed 🙂

Reply:
This is why I am asking whether you use the dependencies anywhere in your code, which would require those who run the library to need them 🙂 (I know they need npm to install this library but to run it is another story)

Reply:
@jonaslagoni  ok